app/static/js/event_list.js

```diff
@@ -67,5 +67,17 @@ function removeRsvpForEvent(eventID){
   })
 }
 
-
-
+// function undoButton(){
+//   undoButton = {from: 'ajax'}
+//   $.ajax({
+//     url: "/event/undo",
+//     type: "POST",
+//     data: undoButton,
+//     success: function(){
+//       alert("Successfully undone!")
+//     },
+//     error: function(error, status){
+//       console.log(error, status)
+//     }
+//   })
+// }
```

Comment:
If we don't need it, let's get rid of it

Reply:
Anna did it.